app/controllers/users_controller.rb

```diff
@@ -1,5 +1,6 @@
 class UsersController < ApplicationController
-  before_action :redirect_to_home
+  before_action :redirect_to_home, only: [:new, :create]
```

Comment:
This method name could be improved. The way it reads, it redirects every user to /home if they try to go to users/new or post to users....I doubt that is really what happens though. I would think about renaming that to describe what it checks for.